test/ca_bundle/ca_bundle_test.go

```diff
@@ -58,28 +59,35 @@ func TestBundle(t *testing.T) {
 
 	parameters := []input{
 		//Use the system pem ca bundle  + local stack pem file ssl should connect thus target string not found
-		{dataInput: "resources/integration/ssl/with/combine/bundle", findTarget: false},
+		{dataInput: "resources/with/combine/bundle", findTarget: false, testType: "metric"},
+		{dataInput: "resources/with/combine/bundle", findTarget: false, testType: "emf"},
 		//Do not look for ca bundle with http connection should connect thus target string not found
-		{dataInput: "resources/integration/ssl/without/bundle/http", findTarget: false},
+		{dataInput: "resources/without/bundle/http", findTarget: false, testType: "metric"},
+		{dataInput: "resources/without/bundle/http", findTarget: false, testType: "emf"},
 		//Use the system pem ca bundle ssl should not connect thus target string found
-		{dataInput: "resources/integration/ssl/with/original/bundle", findTarget: true},
+		{dataInput: "resources/with/original/bundle", findTarget: true, testType: "metric"},
+		{dataInput: "resources/with/original/bundle", findTarget: true, testType: "emf"},
 		//Do not look for ca bundle should not connect thus target string found
-		{dataInput: "resources/integration/ssl/without/bundle", findTarget: true},
+		{dataInput: "resources/without/bundle", findTarget: true, testType: "metric"},
+		{dataInput: "resources/without/bundle", findTarget: true, testType: "emf"},
 	}
 
 	for _, parameter := range parameters {
 		//before test run
-		log.Printf("resource file location %s find target %t", parameter.dataInput, parameter.findTarget)
-		t.Run(fmt.Sprintf("resource file location %s find target %t", parameter.dataInput, parameter.findTarget), func(t *testing.T) {
-			common.ReplaceLocalStackHostName(parameter.dataInput + configJSON)
-			t.Logf("config file after localstack host replace %s", string(readFile(parameter.dataInput+configJSON)))
-			common.CopyFile(parameter.dataInput+configJSON, configOutputPath)
-			common.CopyFile(parameter.dataInput+commonConfigTOML, commonConfigOutputPath)
+		pathDir := parameter.dataInput + "/" + parameter.testType
+		log.Printf("resource file location %s find target %t", pathDir, parameter.findTarget)
+		t.Run(fmt.Sprintf("resource file location %s find target %t", pathDir, parameter.findTarget), func(t *testing.T) {
+			common.RecreateAgentLogfile(logfile)
```

Comment:
Do we need to recreate it? Can't we just delete it? The agent should recreate it on start up.

Reply:
No the agent does not do that on start up. If there is no log file it will not start.

Reply:
The agent will create a new log file if it doesn't exist on start up. I've tested this.

>Logger opens or creates the logfile on first Write.

https://github.com/natefinch/lumberjack

Reply:
I remember this being an issue in the past. When was lumberjack added?